mypy/stubtest.py

```diff
@@ -419,6 +419,16 @@ class SubClass(runtime):  # type: ignore[misc]
         # Examples: ctypes.Array, ctypes._SimpleCData
         pass
 
+    # Runtime class might be annotated with `@final`:
+    if getattr(runtime, "__final__", False) and not stub.is_final:
```

Comment:
If the runtime is sufficiently harebrained, this could raise an exception. Not sure how paranoid we are about that in the rest of stubtest, but might be worth adding a try-except here. 

In pyanalyze (which deals with similar concerns) I ended up adding a small collection of "safe" wrappers that catch any exceptions: https://github.com/quora/pyanalyze/blob/master/pyanalyze/safe.py#L52

Reply:
Agreed!